prereq_tools/base.py

```diff
@@ -785,6 +788,13 @@ def _setup_compiler(self):
         env = self.__env.Clone()
         config = Configure(env)
 
+        if compiler == 'covc':
+            covfile = self.__env.get('PREFIX') + "/../../test.cov"
+            if os.path.isfile(covfile):
+                os.remove(covfile)
+            os.system('/opt/BullseyeCoverage/bin/cov01 -1')
```

Comment:
We should use RUNNER.run_commands([command1, command2])

Furthermore, you can use $COV01 in the commands